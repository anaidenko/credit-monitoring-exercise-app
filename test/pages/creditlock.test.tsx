import React from 'react'

import CreditLock from '@/pages/creditlock'

import { render, withTestRouter } from '../testUtils'

describe('Credit Lock page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(withTestRouter(<CreditLock mock={true} />), {})
    expect(asFragment()).toMatchSnapshot()
  })
})
