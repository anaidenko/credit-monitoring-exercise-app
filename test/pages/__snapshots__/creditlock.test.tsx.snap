// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Credit Lock page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    Loading
  </div>
</DocumentFragment>
`;
